daemon/daemon.go

```diff
@@ -836,12 +843,7 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 		return nil, err
 	}
 
-	volumesDriver, err := graphdriver.GetDriver("vfs", config.Root, config.GraphOptions)
-	if err != nil {
-		return nil, err
-	}
-
-	volumes, err := volumes.NewRepository(filepath.Join(config.Root, "volumes"), volumesDriver)
+	volumesDriver, err := local.New(path.Join(config.Root, "volumes"))
```

Comment:
Little inconsistence, this variable called as `volumeDriver`(without `s`) everywhere else